src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -108,6 +109,11 @@ PerfAnalyzer::CreateAnalyzerObjects()
             model_metadata, model_config, params_->model_version,
             params_->bls_composing_models, params_->input_shapes, backend_),
         "failed to create model parser");
+  } else if (params_->kind == cb::BackendKind::OPENAI) {
+    FAIL_IF_ERR(
+        parser_->InitOpenAI(
+            params_->model_name, params_->model_version, params_->batch_size),
```

Comment:
Why take version and batch size? Are these relevant inputs for OpenAI?

Reply:
I do not think version matters. Batch size might. This is also true for some other service kinds. Keeping the pattern the same for now.

Reply:
batch size testing is a follow-on story. If I find issues, we may have to draw a line in the sand and say for MVP it only supports batch size of 1. I need to get this story in so others can test end-to-end while I test and clean up the smaller details.